boards/arm/mec15xxevb_assy6853/doc/index.rst

```diff
@@ -0,0 +1,240 @@
+| 4-5     | 5-6    | 5-6       | 5-6      | 3-4     |
++---------+--------+-----------+----------+---------+
+| 8-9     | 8-9    | 8-9       | 8-9      | 5-6     |
++---------+--------+-----------+----------+---------+
+|         | 11-12  | 11-12     | 11-12    |         |
++---------+--------+-----------+----------+---------+
+|         |        | 14-15     | 14-15    |         |
++---------+--------+-----------+----------+---------+
+|         |        | 17-18     | 20-21    |         |
++---------+--------+-----------+----------+---------+
+
+Jump settings for MEC150x 144WFBGA Socket DC Assy 6801 Rev B1p0
+===============================================================
+
+The jumper configuration explained above covers the base board. Now the CPU
+board requires the following settings.
+
++-------+-------+
+| JP1   | JP3   |
++=======+=======+
+| 1-2   | 2-3   |
++-------+-------+
+
+Programming and Debugging
+*************************
+
+This board comes with a Cortex ETM port which facilitates tracing and debugging
+using a single physical connection.  In addition, it comes with sockets for
+JTAG only sessions.
+
+Flashing
+========
+
+#. Connect the SPI Dongle ASSY 6791 to J36 (SPI dongle) in order to flash and
+   boot from SHD SPI NOR. Then proceed to flash using Dediprog SF100 or a
+   similar tool for flashing SPI chips. Remember that SPI MISO/MOSI are
+   swapped on dediprog headers!
+
+#. Run your favorite terminal program to listen for output. Under Linux the
+   terminal should be :code:`/dev/ttyACM0`. For example:
+
+   .. code-block:: console
+
+      $ minicom -D /dev/ttyACM0 -o
+
+   The -o option tells minicom not to send the modem initialization
+   string. Connection should be configured as follows:
+
+   - Speed: 115200
+   - Data: 8 bits
+   - Parity: None
+   - Stop bits: 1
+
+#. Connect the MEC15xxEVB_ASSY_6853 board to your host computer using the
+   UART0 port. Then build :ref: `hello_world` application. It is important
+   to generate a binary with a new load address, for example do the following::
+
+        ${OBJCOPY} --change-addresses -0xb0000 -O binary -S ${in_elf} ${out_bin}
+
+   Once you obtain the binary, proceed to use the microchip tool mec2016_spi_gen
+   in order to create the final binary. This binary is what you need to flash
+   in your spi nor.
+
+   .. zephyr-app-commands::
+      :zephyr-app: samples/hello_world
+      :board: mec15xxevb_assy6853
+      :goals: build flash
+
+   You should see "Hello World! mec15xxevb_assy6853" in your terminal.
+
+Debugging
+=========
+
+You can debug an application in the usual way.  Here is an example for the
+:ref:`hello_world` application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: mec15xxevb_assy6853
+   :maybe-skip-config:
+   :goals: debug
+
+References
+**********
+.. target-notes::
+
+.. _MEC1501 Reference Manual:
+    https://github.com/MicrochipTech/CPGZephyrDocs.git
```

Comment:
This might be a better link: https://github.com/MicrochipTech/CPGZephyrDocs/blob/master/MEC1501/MEC1501_Datasheet.pdf
Is this a temporary location for this document?  Will it be here when the product is actually launched, or on the Microchip website itself?